src/javascript/crypto/e2e/extension/ui/dialogs/backupkey/backupkey_test.js

```diff
@@ -87,7 +88,7 @@ function testGetKeyCode() {
   asyncTestCase.waitForAsync('Waiting for dialog to be populated with key');
   dialog.getBackupCode_().addCallback(function(code) {
     assertEquals(code, goog.crypt.base64.encodeByteArray(
-        [1, 1, 2, 3, 4, 5]));
+        [1, 1, 1, 2, 3, 4, 5, 0x8D, 0x87]));
```

Comment:
It isn't compatible yet. There's no code (on either side) to parse the other.  This simply adds the basic e2e code to e2e.  I can add parsing for e2email codes in a separate PR.